src/Tests/Sending/MessageDispatcherTests.cs

```diff
@@ -336,28 +336,6 @@ public async Task Should_allow_mixing_operations()
             Assert.That(someTopicSenderBatchContent, Has.Count.EqualTo(1));
         }
 
-        [Test]
-        public void Should_throw_when_batch_size_exceeded()
```

Comment:
Throwing is automatically done in cases when we fallback to individual sends, and they exceed the quota the service will throw